src/workbench/Workbench.tsx

```diff
@@ -108,6 +127,7 @@ const Workbench = () => {
                 setSerialStateWhenOpen={setSerialStateWhenOpen}
                 setSimulatorShown={setSimulatorShown}
                 simulatorShown={simulatorShown}
+                collapseSidebar={collapseSidebar}
```

Comment:
showSimulator and showSidebar props might be easier to follow than this slightly odd pair

Reply:
I want to rejig this to avoid the sim and the sidebar components knowing about each other. We should be able to pull this behaviour up into Workbench where the layout belongs.